include/swift/AST/NameLookup.h

```diff
@@ -356,18 +356,23 @@ void lookupInModule(ModuleDecl *module, ModuleDecl::AccessPathTy accessPath,
 /// "inherited" by the given declaration at a particular position in the
 /// list of "inherited" types.
 ///
-/// Add anything we find to the
+/// Add anything we find to the \c result vector. If we come across the
+/// AnyObject type, set \c anyObject true.
 void getDirectlyInheritedNominalTypeDecls(
```

Comment:
Would it be cleaner to return a struct or a pair here instead of having two out parameters?